python/lsst/ts/observatory/control/utils/extras/dm_target_catalog.py

```diff
@@ -0,0 +1,146 @@
+# This file is part of ts_observatory_control.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = [
+    "DM_STACK_AVAILABLE",
+    "find_target_radec",
+]
+
+import typing
+
+import astropy.coordinates
+import astropy.units
+import numpy as np
+
+DM_STACK_AVAILABLE = True
+try:
+    from lsst.daf.butler import DeferredDatasetHandle
+    from lsst.geom import SpherePoint
+    from lsst.meas.algorithms import ReferenceObjectLoader
+    from lsst.sphgeom import Angle, Vector3d
+    from lsst.summit.utils import BestEffortIsr
+except ImportError:
+    DM_STACK_AVAILABLE = False
+    import warnings
+
+    warnings.warn("Cannot import required libraries. Module will not work.")
+
+
+def get_reference_object_loader_from_butler(
+    butler: typing.Any,
+    refcat_name: str,
+    config: typing.Any = None,
```

Comment:
FYI this will fail with the new version of MyPy. It should be `config: typing.Any | None = None`.